scripts/update-electron.js

```diff
@@ -11,9 +11,13 @@ async function cleanAndBootstrap(electronVersion) {
     await runInDir('rm -Rf node_modules');
     const packageJsonBkp = fs.readFileSync('./package.json');
     await runInDir('npm i');
-    await runInDir(`npm i electron@${electronVersion}`); // make sure electron is hoisted on the root
+    // Make sure electron is hoisted on the root
+    await runInDir(`npm i electron@${electronVersion}`);
     await runInDir('npm run bootstrap');
     fs.writeFileSync('./package.json', packageJsonBkp);
+    // Run install again to remove root level electron is removed from
```

Comment:
`remove root level electron is removed` :P

Reply:
Whoops, thanks!